.gitlab/ci/.gitlab-ci.yml

```diff
@@ -42,6 +42,7 @@ variables:
   DEMISTO_CONNECTION_POOL_MAXSIZE: "180" # see this issue for more info https://github.com/demisto/etc/issues/36886
   SDK_REF: "master" # The default sdk branch to use
   OVERRIDE_ALL_PACKS: "false"
+  TEST_UPLOAD: "false"
```

Comment:
use it here with true we will add this as false in the upload trigger (and schedule) so default will be test.